protocol/avdl/chat1/local.avdl

```diff
@@ -22,7 +22,10 @@ protocol local {
   }
 
   record MessageAttachment {
-    string path;
+    string storageType;  // where the attachment object is stored (i.e. s3)
```

Comment:
I wonder if `MessageAttachment` should be a `variant`.  We can alternatively have `ATTACHMENT` v `ATTACHMENT_V2` in the variant one level up